content/desktop/release-notes.md

```diff
@@ -24,6 +24,79 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 
+
+{{< release-date date="2024-02-26" >}}
+
+{{< desktop-install all=true version="4.29.0" build_path="/TBD/" >}}
+
+### New
+
+- Beta version of air-gapped containers
+- Added the ability to enforce Rosetta usage via Settings Management
+- Docker socket mount restrictions admin configuration with ECI
+- Moby 26 engine including Buildkit 0.13, sub volumes mounts, networking updates, and improvements to the containerd multi-platform image store UX
+- New Docker Desktop error screens: swift troubleshooting, easy diagnostics uploads, and actionable remediation.
+- Compose now supports Synchronised file shares
+- New interactive Compose CLI
+- Added support for host networking in Docker Desktop (Beta)
+
```

Comment:
Two additional ones
- Beta - Docker Debug for running containers in GUI
- Beta - Volumes Backup & Share extension functionality available in Volumes tab

Reply:
should this be added to the release highlights as well?